zerver/management/commands/deactivate_realm.py

```diff
@@ -25,7 +25,7 @@ def handle(self, *args: Any, **options: str) -> None:
 
         if realm.deactivated:
             print("The realm", options["realm_id"], "is already deactivated.")
-            exit(0)
+            return
```

Comment:
Hmm should these be doing `raise CommandError()` instead of the print? I guess that is a nonzero exit status, so would be a functional change.